package.json

```diff
@@ -4,9 +4,9 @@
   "scripts": {
     "clean": "tsc --build --clean tsconfig.json",
     "clean:all": "yarn clean && nx reset",
-    "prepare": "nx run-many --target=prepare --all --verbose",
-    "build:libs": "nx run-many -t build --projects=tag:type:lib --verbose",
-    "build:apps": "nx run-many -t build --projects=tag:scope:extension --verbose",
+    "prepare": "nx run-many --target=prepare --all --verbose --skip-nx-cache",
```

Comment:
skipping the nx cache defeats the whole purpose

Reply:
if you want to skip the nx cache locally on your machine then you could set the env value `NX_SKIP_NX_CACHE=true`

Reply:
without adding this flag by default to all the build commands

Reply:
oops, the package.json shouldn't be pushed, everyone is handling this on their own. reverting